pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -130,10 +131,8 @@ func affinityAssistantStatefulSet(name string, pr *v1beta1.PipelineRun, claimNam
 	}
 
 	containers := []corev1.Container{{
-		Name: "affinity-assistant",
-
-		//TODO(#2640) We may want to create a custom, minimal binary
-		Image: "nginx",
```

Comment:
it seems like this doesn't completely resolve 2640 - would it make sense to keep this comment, maybe on the line where we default the value of `affinityAssistantImage`?

Reply:
The comment might be put in `controller.yaml` yeah :upside_down_face: